Site-Reliability-Engineering/README.md

```diff
@@ -0,0 +1,46 @@
+# Site Reliability Engineering Roles
+
+Our site reliability engineering roles are defined by expectations for each level in terms of::
+-   **Planning horizon**: The typical timeframe to be thinking ahead about your impacts
+-   **Impact radius**: The number of people or teams typically impacted by your work
+-   **Evaluated by**: The person typically accountable for evaluating your performance
+-   **Responsibility and direction needed**: The expected degree of autonomy at that level
+
+...and in each of the following dimensions:
+
+-   🦉 Domain expertise
+-   🌱 Teaching and Mentoring
+-   🧭 Culture and Leadership
+-   🏆 Customer Success
+
+The following pages provide demonstrable values and concrete examples to describe these expectations:
+
+1. [L2: Site Reliability Engineer](L2-Site-Reliability-Engineer.md)
+2. [L3: Senior Site Reliability Engineer](L3-Senior-Site-Reliability-Engineer.md)
+
+💡 Tip: To help guide conversations about your progression with your manager, open multiple tabs and contrast the expectations across adjacent levels.
+
+See also: guidance on [Deriving an overall rating (Maturing, Performing, Exceeding)](../Software-Engineering/Overall-Ratings.md).
+
+
+# What are SREs?
+SREs understand software systems are not just a pile of code. They understand its purpose, the 
+machinery used to run it, how it\'s managed and that reliable operations require humans to 
+collaborate rather than individual heroism.
+
+They help teams to make the right reliability trade offs. They understand how to measure how well
+ a system is running, identify the high leverage points and move them to improve the outcomes, 
+ costs and/or manage risks. 
+
+They are our resident experts in learning, both from what works and doesn\'t, are unfazed 
+by chaos, they will calmly and practically get problems fixed.
```

Comment:
Liam proposed some rewording of this sentence and another one below. All looks great, once those changes are made. 👍

Reply:
How does this look now?

Reply:
LGTM